sdks/python/apache_beam/io/filesystem.py

```diff
@@ -421,27 +421,44 @@ def __exit__(self, exception_type, exception_value, traceback):
 
 
 class FileMetadata(object):
-  """Metadata about a file path that is the output of FileSystem.match."""
-  def __init__(self, path, size_in_bytes):
+  """Metadata about a file path that is the output of FileSystem.match.
+
+  Fields:
+    path: [Required] file path.
+    size_in_bytes: [Required] file size in bytes.
+    last_updated_in_seconds: [Optional] last modified timestamp of the file, or
+    valued 0.0 if not specified.
```

Comment:
Does a timestamp in seconds match what we have in java? I would expect it to be millis since epoch generally

Reply:
Here it used the same unit with the existing filesystem.last_updated return value. It may be a language-specific convention: scalar timestamp in python appears in float seconds such as `datetime.time.time()`, `os.path.getmtime(dir)`; while in java appears in long milliseconds such as `Date.getTime()`.

Reply:
Lets ask in the next catch up which direction is better